src/app/containers/InlineLink/index.jsx

```diff
@@ -25,8 +25,17 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
   if (result) {
     // the path is the first item in the array
     const path = result[0];
+    const { hash = null } = new URL(locator);
+    const handleClick = () => {
```

Comment:
is this an animated scroll? is animation a requirement?

Reply:
Great observation. It's not animated at the moment @jroebu14. Also not a requirement per se but we should definitely add it because it's great for UX

Reply:
The smooth scrolling functionality seems to be [a little jerky on Safari and IE11 ](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#browser_compatibility). I have tested it on both browsers and it's not working.

Reply:
Is this just standard anchor scroll behaviour?